File: Quorum/solve/cpp/QuorumSolver.cpp
#include <fstream>
#include <topcoder>
#include "Quorum.cpp"
namespace tc = TopCoder;

int main(int argc, char const *argv[]) {
  try {
    std::ifstream ifs(argv[1]);
    vector<int> arr; tc::read(ifs, arr); tc::next(ifs);
    int k; tc::read(ifs, k);
    ifs.close();

    std::ofstream ofs(argv[2]);
    Quorum solver;
    tc::show(ofs, solver.count(arr, k));
    ofs.close();
  } catch (std::exception &e) {
    std::cerr << e.what() << std::endl;
  }
  return 0;
}

File: Quorum/solve/java/QuorumSolver.java
import java.io.FileReader;
import java.io.FileWriter;

import java.util.List;
import java.util.ArrayList;

import org.topcoder.TopcoderReader;
import org.topcoder.TopcoderWriter;
import org.topcoder.TypeRef;

public class QuorumSolver {
    public static void main(String[] args) {
    try {
        TopcoderReader reader = new TopcoderReader(new FileReader(args[0]));
        List<Integer> arrBoxed = (List<Integer>) reader.next(new TypeRef<List<Integer>>(){}.getType());
        int[] arr = new int[arrBoxed.size()];
        for (int _i = 0; _i < arrBoxed.size(); ++_i)
            arr[_i] = arrBoxed.get(_i);
        reader.next();
        
        int k = (Integer) reader.next(Integer.class);
        reader.close();

        Quorum solver = new Quorum();
        TopcoderWriter writer = new TopcoderWriter(new FileWriter(args[1]));
        writer.write(solver.count(arr, k));
        writer.close();
    } catch (Exception err) {
        err.printStackTrace(System.err);
    }
    }
}
